packaging/material/exercise_packaging_part2.md

```diff
@@ -0,0 +1,6 @@
+# Exercise: Packaging Python code Part 2
```

Comment:
The file name should end with `_text.md` such that the GitHub Action can identify it as text file when converting it to PDF.

Why is the exercise split into two files?

Reply:
I split the exercise because there is a clear segregation of tasks in the following way:

1. Preparing the code for packaging
2. Uploading the distribution to a remote packaging index

I think the segregation gives a more structural and modular form to the exercise and also allows the students to understand that these two tasks are really independent. Do you think it is reasonable to keep the two parts?